app/services/katello/pulp/pulp_content_unit.rb

```diff
@@ -0,0 +1,61 @@
+require 'set'
+
+module Katello
+  module Pulp
+    module PulpContentUnit
```

Comment:
I was curious your thinking on making this an composable object instead of a base class the content units inherit from. What prompted me to ask the question was the fact that this module had an initialize method.